WordPress/Classes/Networking/BlogServiceRemoteXMLRPC.m

```diff
@@ -125,4 +181,15 @@ - (WPXMLRPCRequestOperation *)operationForPostFormatsWithBlog:(Blog *)blog
     return operation;
 }
 
+- (RemoteBlogSettings *)remoteBlogSettingFromXMLRPCDictionary:(NSDictionary *)json
+{
+    RemoteBlogSettings *remoteSettings = [[RemoteBlogSettings alloc] init];
+    
+    remoteSettings.name = [json stringForKeyPath:@"blog_title.value"];
+    remoteSettings.desc = [json stringForKeyPath:@"blog_tagline.value"];
+    //remoteSettings.blogPublic = [json numberForKey:@"blog_public"];
```

Comment:
Why is this commented out? What about the other settings (e.g. language)?

I think those belong in a separate PR to begin with, but if we're adding code for them, I think it should be complete.